### PR TITLE
Wrote flight class

### PR DESCRIPTION
**Pull request by justin-obrien-smoothstack**

### CODE REVIEW

**Review by trevorhuis-smoothstack — COMMENTED** [review]
Good stuff, that's as complicated as it gets with Hibernate. It's amazing much easier it makes our lives developing.

File: src/main/java/com/ss/training/utopia/counter/entity/Flight.java [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+	@Id
+	@Column
+	private int departId, arriveId;
+
+	@Id
+	@Column
+	private LocalDateTime departTime;
+
+	@Column
+	private int flightId, seatsAvailable;
```
Comment (sean-darsie) [inline src/main/java/com/ss/training/utopia/counter/entity/Flight.java:31]: I think we should make these 'int'  Integers instead for flexibility.

File: src/main/java/com/ss/training/utopia/counter/entity/Flight.java [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+
+	private static final long serialVersionUID = -8418526716923540697L;
+
+	@Id
+	@Column
+	private int departId, arriveId;
+
+	@Id
+	@Column
+	private LocalDateTime departTime;
```
Comment (sean-darsie) [inline src/main/java/com/ss/training/utopia/counter/entity/Flight.java:28]: change to timestamp

File: src/main/java/com/ss/training/utopia/counter/entity/Flight.java (hunk 1 above)
Comment (justin-obrien-smoothstack) [reply src/main/java/com/ss/training/utopia/counter/entity/Flight.java:31]: That seems like a best practice. I seem to recall having to change an int to an Integer for some reason in some of the entity classes for the LMS. Also it gives me an excuse to overhaul all the data types for optimal memory use and so forth.

**Comment by justin-obrien-smoothstack** [comment]
Hmm, I just pushed to this branch but the commits aren't showing up here, and GitHub is giving me the option to open a PR on the branch even though this one is currently open.

**Comment by justin-obrien-smoothstack** [comment]
Looks like this is a GitHub bug. I'll just have to close this PR and open a new one on the same branch.

**Pull request closed without merging**